<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMenu</class>
 <widget class="QWidget" name="FormMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:none;
	background-color: rgb(95, 95, 95);
}
QPushButton::focus{
	color:rgb(60, 60, 60);
	border:none;
	background-color: rgb(208, 230, 255);
}
QPushButton::disabled{
	color:rgb(0, 0, 0);
	border:none;
	background-color: rgb(226, 226, 226);
}
QLabel{
	color:rgb(255, 236, 213);
	background-color: rgb(63, 54, 50);
}</string>
  </property>
  <widget class="QLabel" name="labelBG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>272</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image/BK.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStationSet">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>60</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Station Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSysStatus">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>125</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Sys Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAudio">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>190</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Audio</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
